scripts/deploy.sh

```diff
@@ -278,9 +300,15 @@ _postinstall_postgres_clr() {
   sudo chown postgres:postgres /var/lib/pgsql/.profile
 }
 
+_postinstall_postgres_centos() {
+  sudo postgresql-setup initdb
+  sudo sed -i 's/ident$/md5/' /var/lib/pgsql/data/pg_hba.conf
+}
+
 _start_postgres() {
   sudo systemctl enable postgresql
   sudo systemctl start postgresql
+  sudo systemctl restart postgresql
```

Comment:
Why do you need to restart Postgres? previous line (#310) is already starting it

Reply:
I got this issue because if postgres is already started, then start command does nothing, but I need it reloaded due to changes in pg_hba.conf, we can do 2 things, remove the start command or remove the restart command but add a stop command first.

Reply:
Removing start should be enough.

Reply:
Yes, looks good